### ISSUE TITLE
rim rstudio <version> does not open rstudio

### ISSUE THREAD

**Issue by eliocamp** [r-lib/rig#77]
I'm on elementary OS 6.1 (Built on Ubuntu 20.04.3 LTS). Running `rig rstudio` with a version number doesn't open rstudio. I get:

```bash
elio@elio:~$ rig rstudio 4.1.3
[INFO] Running xdg-open 4.1.3
````

**Comment by gaborcsardi** [comment]
Thanks, that does seem like a bug.

**Comment by gaborcsardi** [comment]
Fixed in 0.4.1: https://github.com/r-lib/rig/releases/tag/v0.4.1
